adapt/engine.py

```diff
@@ -45,7 +45,7 @@ def __init__(self, tokenizer=None, trie=None):
                 example EnglishTokenizer()
             trie(Trie): tree of matches to Entites
         """
-        pyee.EventEmitter.__init__(self)
+        pyee.BaseEventEmitter.__init__(self)
```

Comment:
Is this change necessary? I didn't have to make the change as part of my PR #106 , and all tests continue to pass.

Reply:
It's not necessary but using EventEmitter gives a depreciation warning.

Reply:
ACK. Happy to take your PR, but it needs to bump version in setup.py.

Reply:
Pushed version bump